docs/en/sql-reference/functions/time-series-functions.md

```diff
@@ -0,0 +1,61 @@
+---
```

Comment:
This file is shown as newly introduced but I think you want to merge from master and add new docs to the existing markdown added with https://github.com/ClickHouse/ClickHouse/pull/56171